festipals/src/samplePals.json

```diff
@@ -0,0 +1,28 @@
+{
+  "pals": [
+    {
+      "_id": "def",
+      "first_name": "Sansa",
+      "last_name": "Stark",
+      "e_mail": "test1@email.com",
+      "pals": ["abc", "ghi"],
+      "acts": ["ijklmnop", "asdf"]
+    },
+    {
+      "_id": "ghi",
+      "first_name": "Sandor the hound",
+      "last_name": "Clegane",
+      "e_mail": "test2@email.com",
+      "pals": ["abc", "def"],
+      "acts": ["ijklmnop", "asdf"]
+    }
+  ],
+  "user": {
```

Comment:
This user should also just be in the pals collection. To have a logged in user we should store a pals id in a session.